src/open_inwoner/scss/admin/_admin_theme.scss

```diff
@@ -330,3 +330,20 @@ form div.help {
   display: inline-block;
   margin-left: inherit !important;
 }
+
+/* Custom code fields */
+.textfield {
+  &--code {
+    font-family: monospace;
+    font-size: 95%;
+  }
+}
+.css-properties-table {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-column-gap: 2em;
+  &__column {
+    font-family: monospace;
+    font-size: 95%;
```

Comment:
What's the reason for this 5% decrease? I can barely tell the difference from looking at the screen.

Reply:
To make it slightly smaller because monospace is so chonky.